pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -184,6 +193,52 @@ func createCNPCRD(clientset apiextensionsclient.Interface) error {
 	return createUpdateCRD(clientset, "CiliumNetworkPolicy/v2", res)
 }
 
+// createCGNPCRD creates and updates the CiliumGlobalNetworkPolicies CRD. It should be called
+// on agent startup but is idempotent and safe to call again.
+func createCCNPCRD(clientset apiextensionsclient.Interface) error {
+	var (
+		// CustomResourceDefinitionSingularName is the singular name of custom resource definition
+		CustomResourceDefinitionSingularName = "ciliumclusterwidenetworkpolicy"
+
+		// CustomResourceDefinitionPluralName is the plural name of custom resource definition
+		CustomResourceDefinitionPluralName = "ciliumclusterwidenetworkpolicies"
+
+		// CustomResourceDefinitionShortNames are the abbreviated names to refer to this CRD's instances
+		CustomResourceDefinitionShortNames = []string{"ccnp", "ciliumcnp"}
```

Comment:
I don't think `ciliumcnp` is necessary here - CNP is almost always used to abbeviate `CiliumNetworkPolicy`, while here it refers to `ClusterwideNetworkPolicy`. I'd prefer we keep only `ccnp`.